app/src/main/java/com/example/easysurvey/MainActivity.kt

```diff
@@ -56,7 +56,7 @@ fun HomepageButtons() {
 
 @Preview(showBackground = true)
 @Composable
-fun DefaultPreview() {
+fun defaultPreview() {
```

Comment:
changed function name to lowercase camelcase per the suggestions here: https://kotlinlang.org/docs/coding-conventions.html#source-file-organization

Reply:
I really want to agree with you on this, as it is normally the case, but composable functions should be nouns, and PascalCase, believe it or not.  
https://developer.android.com/kotlin/style-guide